app_dart/lib/src/foundation/utils.dart

```diff
@@ -102,6 +102,13 @@ FutureOr<String> getUrl(
   }
 }
 
+/// Expands globs string to a regex for evaluation.
+Future<RegExp> parseGlob(String glob) async {
+  glob = glob.replaceAll('**', '[A-Za-z0-9_/.]+');
+  glob = glob.replaceAll('*', '[A-Za-z0-9_.]+');
+  return RegExp('^$glob\$');
+}
+
 /// Returns a LUCI [builder] list that covers changed [files].
```

Comment:
Update this doc to include the run_if_not option.

Reply:
Updated!